prbench-bilevel-planning/tests/env_models/geom2d/test_clutteredretrieval2d.py

```diff
@@ -210,44 +208,6 @@ def test_clutteredretrieval2d_skills():
         in abstract_state1.atoms
     )
 
-    # Placing the obstruction to empty place.
```

Comment:
Is it possible to specify the sample for the unit skill here, just for the unit test?